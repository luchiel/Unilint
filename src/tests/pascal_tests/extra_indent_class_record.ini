[indentation]
indentation_style = consistent
indentation_policy = consistent
extra_indent_for_blocks = consistent

[spaces_and_newlines]
start_block_at_newline = consistent
