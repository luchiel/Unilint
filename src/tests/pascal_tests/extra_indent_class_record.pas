type
  Bet =
    record
      money_on, money_against: integer;
    end;
  
  Point = class
      x, y: integer;
    end;

  Square = record
    a, b, c, d: Point;
  end;