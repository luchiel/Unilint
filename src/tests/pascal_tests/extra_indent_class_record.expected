12:1: indent too small
13:1: indent too small
